test/models/proquest_export_batch_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class ProquestExportBatchTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
It might be useful to confirm the json builds correctly here? Not sure.

Reply:
Just as I try to introduce a little danger into our test suite... (I totally forgot to write any tests for this model; thanks for catching!)